app/src/app/api/v0/city/[city]/user/route.ts

```diff
@@ -11,14 +11,32 @@ export const POST = apiHandler(async (_req: NextRequest, { params }) => {
     userId: randomUUID(),
     ...body,
   });
-  const cityId = params.city;
 
   const city = await db.models.City.findOne({ where: { locode: params.city } });
   if (!city) {
     throw new createHttpError.NotFound("City not found");
   }
 
-  user.addCity(cityId);
+  user.addCity(city.cityId);
 
   return NextResponse.json({ data: user });
 });
+
+export const GET = apiHandler(async (_req: NextRequest, { params }) => {
+  const users = await db.models.User.findAll({
```

Comment:
This API route doesn't use params.user, so why is it in this file instead of the file one level above? Also, both GET functions seem to do the same thing?